roles/scp_http_int/tasks/user_creation.yml

```diff
@@ -0,0 +1,6 @@
+---
+- user: name=logs state=present
```

Comment:
I tend to prefer using _ before system users, since that will prevent problem in the future. And I would also prefer something more verbose than "logs", like, "logs-uploader" or something that tell exactly what the server is used for.

Reply:
And using a variable too, so we do not repeat the same name in various place.